docs/docs/troubleshooting.md

```diff
@@ -79,4 +79,14 @@ to avoid accidental bonding between the halves.
 1. Flash the actual image for each half of the split keyboard (e.g `my_board_left.uf2` to the left half, `my_board_right.uf2` to the right half)
 
 After completing these steps, pair the halves of the split keyboard together by resetting them at the same time. Most commonly, this is done by grounding the reset pins
-for each of your keyboard's microcontrollers or pressing the reset buttons at the same time.
\ No newline at end of file
+for each of your keyboard's microcontrollers or pressing the reset buttons at the same time.
+
+### Connectivity Issues ###
+
+Some users may experience poor connection from their keyboard to their host device, whether it be from a bad Bluetooth card, a metal enclosure on the keyboard or host, or distance to the host device. Increasing the transmit power of your board may solve this problem. To do this, simply enable the `CONFIG_BT_CTLR_TX_PWR_PLUS_8`  configuration value in the `.conf` file of your user config directory as such:
```

Comment:
> Increasing the transmit power of your board may solve this problem.

Increasing the transmit power of the keyboard's BLE radio may alleviate this problem.